bytecomp/bytepackager.ml

```diff
@@ -20,31 +20,43 @@ open Misc
 open Instruct
 open Cmo_format
 module String = Misc.Stdlib.String
+module GlobalMap = Symtable.GlobalIds.Map
```

Comment:
it looks like additionally defining `GlobalSet` would make some parts of the patch below lighter